RecoLocalCalo/HcalRecAlgos/src/MahiFit.cc

```diff
@@ -292,7 +292,6 @@ void MahiFit::updatePulseShape(double itQ, FullSampleVector &pulseShape, FullSam
       double tmp = 0.5*( nnlsWork_.pulseP[iTS+delta]*nnlsWork_.pulseP[jTS+delta] +
```

Comment:
as mentioned already, what about the pulseDeriv 10 lines above? Should it be fixed as well to actually be a derivative.
pulseM and pulseP are converted to residuals only after pulseDeriv assignment call is made.
This apparently only affects the hit time result.